modules/end-user-guide/examples/snip_che-configuring-a-workspace-to-use-an-intellij-based-ide.adoc

```diff
@@ -0,0 +1,65 @@
+.Prerequisites
+
+* Define the IntelliJ-based IDE container image to use: __<IntelliJ-container-image>__. It can be:
+** A prebuilt IntelliJ-based IDEs image:
++
+`quay.io/che-incubator/che-idea:latest`:: IntelliJ IDEA Community Edition
++
+`quay.io/che-incubator/che-pycharm:latest`:: PyCharm Community Edition
+** A custom editor image, see xref:building-intellij-editor-images.adoc[].
+
+.Procedure
+
+. In the root directory of the workspace, create a `+meta.yaml+` file with the following content:
++
+[source,yaml,subs="+quotes,macros,attributes"]
+----
+apiVersion: v2
+publisher: <username>
+name: intellij-ide
+version: latest
+type: Che Editor
+displayName: IntelliJ-based IDE
+title: IntelliJ-based IDE as Editor for Eclipse Che
+description: IntelliJ-based IDE running using Projector
+icon: https://www.jetbrains.com/apple-touch-icon.png
+category: Editor
+repository: https://github.com/che-incubator/jetbrains-editor-images
+firstPublicationDate: "2021-04-10"
+spec:
+  endpoints:
+   -  name: "intellij"
+      public: true
+      targetPort: 8887
+      attributes:
+        protocol: http
+        type: ide
+        path: /projector-client/index.html?backgroundColor=434343&wss
+  containers:
+   - name: intellij-ide
+     image: "__<IntelliJ-container-image>__"
+     mountSources: true
+     volumes:
+         - mountPath: "/home/projector-user"
+           name: projector-user
+     ports:
+         - exposedPort: 8887
+     memoryLimit: "4096M"
+----
+
+. Add the following component to the workspace devfile:
++
+[source,yaml]
+----
+components:
+ - type: cheEditor
+   reference: meta.yaml
```

Comment:
I committed a branch with the meta.yaml and this devfile modification, but I got this error:

```
Danger alert:Workspace che-docs-ivmy9 failed to start. Failed to start the workspace che-docs-ivmy9, reason: Plugin reference URL 'meta.yaml' is invalid.
```

I don't find examples to put an accurate content for the `reference` key. Can you help?

test it: https://workspaces.openshift.com#https://github.com/themr0c/che-docs/tree/test-intellij

Reply:
@vzhukovs could you clarify which version of Eclipse Che / CRW is expected to be used for verification? workspaces.openshift.com is running against 2.10.1, and I suspect that this functionality is not supposed to work there?

Reply:
@ibuziuk if `reference` section points to the editor configuration by the url. I mean, that user uploaded somewhere `meta.yaml`, then it is no difference on what exact version of Eclipse Che/CRW it can be tested. But in context of CRW it will be provided on 2.11. And in context of Eclipse Che, plugin-registry will be updated according to this issue: https://github.com/eclipse/che/issues/19596 soon. I suppose, it will be available in 7.36